### DISCUSSION TITLE
v19.0.0

### DISCUSSION THREAD

**Discussion by arturovt** [ngxs/store#2275] [Releases]
## What's Changed
* Update schematics.md by @domlija in https://github.com/ngxs/store/pull/2271
* build: add Angular 19 support by @arturovt in https://github.com/ngxs/store/pull/2269
* chore: release v19.0.0 by @arturovt in https://github.com/ngxs/store/pull/2274

## New Contributors
* @domlija made their first contribution in https://github.com/ngxs/store/pull/2271

**Full Changelog**: https://github.com/ngxs/store/compare/v18.1.6...v19.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ngxs/store/releases/tag/v19.0.0'>v19.0.0</a>.</em>

**Comment by dinbtechit** [comment]
Upgraded the project to Angular 19 and NGXS 19.  Creating a store using NGXS CLI gives me the following error. I don't see a package called  `@angular-devkit/core`...

```
/home/dinesh/.nvm/versions/node/v18.20.6/bin/node /home/dinesh/code/my-resume/node_modules/@angular/cli/bin/ng.js generate @ngxs/store:store --name test --path \app\store --project my-resume
An unhandled exception occurred: Cannot find module '@angular-devkit/core'
Require stack:
- /home/dinesh/code/my-resume/node_modules/@ngxs/store/schematics/src/utils/generate-utils.js
- /home/dinesh/code/my-resume/node_modules/@ngxs/store/schematics/src/store/store.factory.js
- /home/dinesh/code/my-resume/node_modules/@angular-devkit/schematics/tools/export-ref.js
- /home/dinesh/code/my-resume/node_modules/@angular-devkit/schematics/tools/index.js
- /home/dinesh/code/my-resume/node_modules/@angular/cli/src/command-builder/schematics-command-module.js
- /home/dinesh/code/my-resume/node_modules/@angular/cli/src/commands/generate/cli.js
- /home/dinesh/code/my-resume/node_modules/@angular/cli/src/commands/command-config.js
- /home/dinesh/code/my-resume/node_modules/@angular/cli/src/command-builder/command-runner.js
- /home/dinesh/code/my-resume/node_modules/@angular/cli/lib/cli/index.js
- /home/dinesh/code/my-resume/node_modules/@angular/cli/lib/init.js
See "/tmp/ng-EIaoFw/angular-errors.log" for further details.
Done

```

**Reply by dinbtechit** [reply]
I was able to resolve the error by installing `npm install --save-dev @angular-devkit/core` strange I never had this package installed. I also migration from Angular 15 to 19. ofcourse incrementally. For Some reasons that package was never there.
